### PR TITLE
Write errors to stderr and exit with error code 1

### PR DESCRIPTION
**Pull request by DavidTranDucVL**
Related issue: https://github.com/vacuumlabs/cardano-hw-cli/issues/23

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -45,5 +45,5 @@ const executeCommand = async (): Promise<void> => {
 }
 
 executeCommand().catch((e: Error) => {
-  console.log('Error:', e.message)
-}).finally(() => process.exit())
+  console.error('Error:', e.message)
+}).finally(() => process.exit(1))
```
Comment (PeterBenc) [inline src/index.ts:49]: Why exit with 1?
https://stackoverflow.com/questions/43147330/what-is-difference-between-method-process-exit1-and-process-exit0-in-node-js

Comment (DavidTranDucVL) [reply src/index.ts:49]: It's explained in the issue https://github.com/vacuumlabs/cardano-hw-cli/issues/23

Comment (PeterBenc) [reply src/index.ts:49]: I see, but still, shouldnt it do `.then(() => process.exit(0)).catch((e) => process.exit(1))`

**Review by PeterBenc — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (DavidTranDucVL) [reply src/index.ts]: Yes, I made mistake, fixed.

**Pull request merged**